Documentation/dev/libvirt-howto.md

```diff
@@ -7,16 +7,44 @@ for operator development.
 ### 1. One-time setup
 It's expected that you will create and destroy clusters often in the course of development. These steps only need to be run once (or once per RHCOS update).
 
-#### 1.1 Pick a name and ip range
+#### 1.1 Install Dependencies and Enable Libvirt
+First install some dependencies needed to complete these instructions and then
+enable the libvirt daemon.
+
+```sh
+sudo dnf copr enable vbatts/bazel
```

Comment:
This is not libvirt-specific; I'd rather have it somewhere else (`CONTRIBUTING.md`?  A new `Documentation/dev/dependencies.md`?).  Also, for folks not using `dnf` (e.g. me on my RHEL 7.5 CSB laptop), a link to [here][1] gives you instructions for other systems (e.g. the release repo uses [this][2]).

[1]: https://copr.fedorainfracloud.org/coprs/vbatts/bazel/
[2]: https://github.com/openshift/release/blob/158650821db4561f2896f4ce3456fd2934d88100/projects/origin-release/bazel/Dockerfile#L6

Reply:
Good point.  I'll move the stuff that isn't libvirt-howto specific somewhere else.  I guess I'll create a dependencies.md file unless someone has a strong opinion otherwise.